app/views/partials/_post_comments.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="comments-card info-card">
+<% post.recent_comments.reverse.each do |comment| %> 
+  <p><%= comment.user.name %>: <%= comment.text %></p>
```

Comment:
- [Optional] Kindly ensure that all the comments (not recent_comments) are rendered in the post_show page.